### PR TITLE
Added Proxmox LXC + Nvidia (Manual) to quickstart.adoc

### PR DESCRIPTION
**Pull request by vinibutturi**
Added a section to quickstart on how to setup Wolf on Proxmox LXC + Nvidia (Manual)

### CODE REVIEW

**Comment by brunnels** [comment]
Thanks @vinibutturi

I'm working on implementing this today so perfect timing!

**Comment by vinibutturi** [comment]
> I'm working on implementing this today so perfect timing!

let us know how it goes and if we need to add anything extra....

File: docs/modules/user/pages/quickstart.adoc [hunk 1]
```diff
@@ -364,6 +364,255 @@ docker run \
...
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|   No running processes found                                                            |
++-----------------------------------------------------------------------------------------+
+```
+
+`ls -l /dev/nividia*`
```
Comment (ABeltramo) [inline docs/modules/user/pages/quickstart.adoc]: I guess it's a typo `/dev/nividia*` should be `/dev/nvidia*`

File: docs/modules/user/pages/quickstart.adoc [hunk 2]
```diff
@@ -364,6 +364,255 @@ docker run \
...
+      - /dev/nvidia-modeset
+    device_cgroup_rules:
+      - 'c 13:* rmw'
+    network_mode: host
+    restart: always
+
+volumes:
+  nvidia-driver-vol:
+    external: true
+```
```
Comment (ABeltramo) [inline docs/modules/user/pages/quickstart.adoc]: This is the same as the normal guide, right? Can't we just say that from this point on you can follow the guide on the other tabs?

Comment (vinibutturi) [reply docs/modules/user/pages/quickstart.adoc]: pretty much, I'll do that and just add a couple remarks

File: docs/modules/user/pages/quickstart.adoc (hunk 1 above)
Comment (vinibutturi) [reply docs/modules/user/pages/quickstart.adoc]: thanks, will fix.

File: docs/modules/user/pages/quickstart.adoc [hunk 3]
```diff
@@ -364,6 +365,193 @@ docker run \
...
+====
+
+The easiest way to make the change persistent is to add `nvidia-drm.modeset=1` to the `GRUB_CMDLINE_LINUX_DEFAULT` line in `/etc/default/grub` ex:
+
+....
+GRUB_CMDLINE_LINUX_DEFAULT="quiet nvidia-drm.modeset=1"
+....
+
+Then `sudo update-grub` and `reboot`.
+====
```
Comment (ABeltramo) [inline docs/modules/user/pages/quickstart.adoc]: I'm sorry to be a pain in the ass (and I'd be happy to do the edit myself if you don't want to) but if we remove this section wouldn't this tab be valid for all LXC setups no matter which GPU they've got?  

The instructions below are pretty good to get a LXC container up and running, then we just say to complete the installation based on the GPU following the other tabs. 

The udev rules and the kernel load stuff should probably be added to the Nvidia tab as well, they might get handy if some OS doesn't include those.

Comment (vinibutturi) [reply docs/modules/user/pages/quickstart.adoc]: hahahaha no worries. I see what you mean, it makes sense.

I will remove the top part and just add a small note saying: You should make sure you have the proper GPU drivers and input udev rules on your PVE host before proceeding and will move the nvidia udev rules into the nvidia tabs.

**Comment by ABeltramo** [comment]
Nice, I like it! Thank you so much for your contribution!

**Pull request merged**